packages/@dcl/ecs/src/components/extended/InputModifier.ts

```diff
@@ -0,0 +1,59 @@
+import { IEngine, LastWriteWinElementSetComponentDefinition, Entity } from '../../engine'
+import { InputModifier, PBInputModifier_StandardInput, PBInputModifier } from '../generated/index.gen'
+import {} from '../generated/InputModifier.gen'
+
+/**
+ * @public
+ */
+export interface InputModifierHelper {
+  /**
+   * @returns a input modifier mode
+   */
+  Standard: (standard: PBInputModifier_StandardInput) => PBInputModifier['mode']
+}
+
+/**
+ * @public
+ */
+export interface InputModifierComponentDefinitionExtended
+  extends LastWriteWinElementSetComponentDefinition<PBInputModifier> {
+  /**
+   * InputModifier helper with constructor
+   */
+  InputModifier: InputModifierHelper
```

Comment:
I think this should be named in a different way because this way it generates the following code:
```ts
      mode: InputModifier.InputModifier.Standard({ disableAll: true })
```
and its redundant. Maybe it should be `InputModifier.Mode.Standard...`
```suggestion
  Mode: InputModifierModeHelper
```